internal/handlers/common/count.go

```diff
@@ -15,67 +15,37 @@
 package common
 
 import (
-	"fmt"
+	"go.uber.org/zap"
 
-	"github.com/FerretDB/FerretDB/internal/handlers/commonerrors"
+	"github.com/FerretDB/FerretDB/internal/handlers/commonparams"
 	"github.com/FerretDB/FerretDB/internal/types"
 )
 
 // CountParams represents the parameters for the count command.
 type CountParams struct {
-	Filter         *types.Document
-	DB, Collection string
-	Skip, Limit    int64
-}
+	Filter     *types.Document `name:"query"`
```

Comment:
Let's use a less generic tag name – the possibility of clashes is very high with them

Reply:
Changed it to `ferretdb`.